### PR TITLE
WIP - Audio Renderer

### PR DESCRIPTION
**Pull request by jtamiace**
## Summary

Audio renderer for Kolibri. Will continue to update this PR with commits that add functionality to the elements.

## TODO

Higher priority
- [x] HTML/CSS layout
- [x] Install html5media.js library, which allows HTML5 audio tag to run on all browsers (thanks to @rtibbles )
- [x] Play/pause button works
- [x] Current time / total time shows
- [x] Current time / total time is in XX:XX format
- [x] Round tracker moves across screen as audio plays
- [x] Round tracker can be dragged and updates time
- [x] Time tracker styles are compatible across (reasonably most) browsers

Less priority
- [x] Restart button works
- [x] + 10s button works
- [x] - 10s button works
- [ ] Show buffering progress in time tracker

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Issues addressed

List the issues solved or partly solved by the PR

## Documentation

If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)

## Screenshots (if appropriate)

audio 2

### CODE REVIEW

**Comment by jtamiace** [comment]
what have i done

File: kolibri/core/templates/kolibri/base.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% load i18n kolibri_tags webpack_tags js_reverse cache %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
```
Comment (jtamiace) [inline kolibri/core/templates/kolibri/base.html:4]: @rtibbles is it ok that i added this meta tag in the base.html file, or should this tuck away somewhere else?

Comment (rtibbles) [inline kolibri/core/templates/kolibri/base.html:4]: Not sure about the properties of this tag, so let's leave it here for now - but we may want to add it dynamically from within the audio component.

**Comment by jtamiace** [comment]
@rtibbles @indirectlylit audio player is fully functional now if you want to take a look. 

currently going to continue working on cross-browser compatible styling

File: kolibri/core/templates/kolibri/base.html (hunk 1 above)
Comment (rtibbles) [inline kolibri/core/templates/kolibri/base.html:4]: My further research indicates that, yes, we definitely want this here :)

File: kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue [hunk 2]
```diff
@@ -30,44 +30,72 @@
...
+        sec: 0,
+      },
+      totalMinutes: {
+        type: Number,
+        totalmin: 0,
+      },
+      totalSeconds: {
+        type: Number,
+        totalsec: 0,
+      },
```
Comment (indirectlylit) [inline kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue]: Hey - looks like you may be applying [props-style validation](https://vuejs.org/guide/components.html#Prop-Validation) to the `data` attributes? I don't believe it works the same way for `data` - as far as I know vue doesn't type-check or validate these values.

Comment (jtamiace) [inline kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue]: ah, ok, reverting back! (i had thought it applied to data as well based on yesterday's session)

Comment (indirectlylit) [inline kolibri/plugins/audio_mp3_render/assets/src/vue/index.vue]: My bad! didn't mean to imply that

**Comment by jtamiace** [comment]
not again...

**Pull request closed without merging**